addons/map_gestures/functions/fnc_initDisplayDiary.sqf

```diff
@@ -0,0 +1,75 @@
+#include "script_component.hpp"
+/*
+ * Author: Dslyecxi, MikeMatrix, joko // Jonas
+ * Bind all required EventHandlers to Player map.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * _mapCtrl call ace_map_gestures_fnc_initDisplayDiary
+ *
+ * Public: No
+ */
+params ["_mapCtrl"];
+TRACE_1("initDisplayDiary",_mapCtrl);
+
+if (!isNil QGVAR(DrawMapHandlerID)) then {
+    _mapCtrl ctrlRemoveEventHandler ["Draw", GVAR(DrawMapHandlerID)];
+    GVAR(DrawMapHandlerID) = nil;
+};
+
+GVAR(DrawMapHandlerID) = _mapCtrl ctrlAddEventHandler ["Draw", { [_this select 0, [ACE_player]] call FUNC(drawMapGestures); }];
+
+// MouseMoving EH.
+if (!isNil QGVAR(MouseMoveHandlerID)) then {
+    _mapCtrl ctrlRemoveEventHandler ["MouseMoving", GVAR(MouseMoveHandlerID)];
+    GVAR(MouseMoveHandlerID) = nil;
+};
+
+GVAR(MouseMoveHandlerID) = _mapCtrl ctrlAddEventHandler ["MouseMoving", {
+    if (!GVAR(enabled)) exitWith {};
+    params ["_control", "_posX", "_posY"];
+    private _position = _control ctrlMapScreenToWorld [_posX, _posY];
+    GVAR(cursorPosition) = _position;
+    // Don't transmit any data if we're using the map tools
+    if (!GVAR(EnableTransmit) || {(["ace_maptools"] call EFUNC(common,isModLoaded)) && {EGVAR(maptools,mapTool_isDragging) || EGVAR(maptools,mapTool_isRotating)}}) exitWith {};
+    if (_position distance2D (ACE_player getVariable [QGVAR(pointPosition), [0, 0, 0]]) >= 1) then {
+        [ACE_player, QGVAR(pointPosition), _position, GVAR(interval)] call EFUNC(common,setVariablePublic);
+    };
+}];
+
+// MouseDown EH
+if (!isNil QGVAR(MouseDownHandlerID)) then {
+    _mapCtrl ctrlRemoveEventHandler ["MouseButtonDown",GVAR(MouseDownHandlerID)];
+    GVAR(MouseDownHandlerID) = nil;
+};
+
+GVAR(MouseDownHandlerID) = _mapCtrl ctrlAddEventHandler ["MouseButtonDown", {
+    if (getClientStateNumber < 10) exitWith {};
```

Comment:
this now runs on all the 4 diarys so we could remove this and it should work on briefing screen, 
which some people might want (maybe another setting)

Reply:
maybe new setting
```
briefing all
briefing friendly
briefing none
```

Reply:
i tested that and it wont. i tried it and the position does not get transmitted correctly. atleast after my tries.